docs/tutorial/beginner.rst

```diff
@@ -0,0 +1,439 @@
+##################################################
+Beginner Tutorial - Set up & Running One Algorithm
+##################################################
+
+TODO: add an explanation of this tutorial
+
+Step 0: Clone the SPRAS repository, set up the environment, and run Docker
+==========================================================================
+
+0.1 Start Docker
+----------------
+
+Launch Docker Desktop and wait until it says "Docker is running".
+
+0.2 Clone the SPRAS repository
+-------------------------------
+
+Visit the `SPRAS github repository <https://github.com/Reed-CompBio/spras>`__ and clone it locally
+
+0.3 Set up the SPRAS environment
+-------------------------------------
+
+From the root directory of the SPRAS repository, create and activate the Conda environment:
+
+.. code:: bash
+
+    conda env create -f environment.yml
+    conda activate spras
+    python -m pip install .
+
+0.4 Test the installation
+-------------------------
+
+Run the following command to confirm that SPRAS has been set up successfully from the command line:
+
+.. code:: bash
+
+   python -c "import spras; print('SPRAS import successful')"
+
+Step 1: Overview of the SPRAS Folder Structure
+==============================================
+
+After cloning SPRAS, you will find four main folders that organize everything needed to run and analyze workflows.
+
+.. code-block:: text
+
+   spras/
+   ├── .snakemake/
+   │   └── log/
+   │       └── ... snakemake log files ...
+   ├── config/
+   │   └── ... other configs ...
+   ├── inputs/
+   │   └── ... input files ...
+   ├── outputs/
+   │   └── ... output files ...
+
+.snakemake/
+-----------
+
+The log/ directory contains records of all Snakemake jobs that were executed for the SPRAS run, including any errors encountered during those runs.
+
+config/
+-------
+
+Holds configuration files (YAML) that define which algorithms to run, what datasets to use, and which analyses to perform.
+
+input/
+------
+
+Contains the input data files, such as interactome edge files and input nodes. This is where you can place your own datasets when running custom experiments.
+
+output/
+-------
+
+Stores all results generated by SPRAS. Subfolders are created automatically for each run, and their structure can be controlled through the configuration file.
+
+By default, the directories are set to config/, input/, and output/. The config/, input/, and output/ folders can be placed anywhere within the SPRAS repository. Their input/ and output/ locations can be updated in the configuration file, and the configuration file itself can be found by providing its path when running SPRAS.
```

Comment:
Can we place these directories anywhere? They don't have to be subdirectories anymore, do they? Do absolute paths work?

Reply:
I have never tried an absolute path to put them anywhere. I thought it was forced to be within spras, but I'm not sure anymore. I'll test it out.